### PR TITLE
docs: switch to dropcss in AMP example

### LINKED ISSUE #10536: Migrate off purify-css
### Describe the bug

`purify-css` has not been updated in 6 years and is causing security warnings in the AMP example

### Reproduction

https://github.com/sveltejs/kit/blob/e133aba479fa9ba0e7f9e71512f5f937f0247e2c/packages/kit/test/apps/amp/src/hooks.server.js#L25

### Logs

_No response_

### System Info

```Shell
`master`
```

### Severity

annoyance

### Additional Information

Some alternatives:
- https://github.com/FullHuman/purgecss
- https://github.com/leeoniya/dropcss

### PR DESCRIPTION
**Pull request by ghostdevv**
Closes #10536

- Switch to using `dropcss`
- Add note about removing css in the docs
- Add note in docs about adding `<html amp>` as we don't do that anywhere. I also want to discourage people from using ⚡ since dropcss [doesn't support it](https://github.com/leeoniya/dropcss/pull/48) - people definitely will be using it so I've added a safeguard (`.replace('⚡', 'amp')`)

### Please don't delete this checklist! Before submitting the PR, please make sure you do the following:
- [x] It's really useful if your PR references an issue where it is discussed ahead of time. In many cases, features are absent for a reason. For large changes, please create an RFC: https://github.com/sveltejs/rfcs #10536
- [x] This message body should clearly illustrate what problems it solves.
- [ ] Ideally, include a test that fails without this PR but passes with it.

### Tests
- [x] Run the tests with `pnpm test` and lint the project with `pnpm lint` and `pnpm check`

### Changesets
- [ ] If your PR makes a change that should be noted in one or more packages' changelogs, generate a changeset by running `pnpm changeset` and following the prompts. Changesets that add features should be `minor` and those that fix bugs should be `patch`. Please prefix changeset messages with `feat:`, `fix:`, or `chore:`.

### CODE REVIEW

**Comment by ghostdevv** [comment]
Don't merge until the site build is fixed :pray:

File: documentation/docs/40-best-practices/20-seo.md [hunk 1]
```diff
@@ -106,21 +106,63 @@ export default config;
...
+			if (done) return amp.transform(buffer);
 		}
 	});
 }
 ```
 
+To prevent shipping any unused CSS as a result of transforming the page to amp, we can use [`dropcss`](https://www.npmjs.com/package/dropcss):
+
+```js
+/// file: src/hooks.server.js
```
Comment (PuruVJ) [inline documentation/docs/40-best-practices/20-seo.md:137]:
```suggestion
/// file: src/hooks.server.js
// @errors: 2307
```

should fix it for now

Comment (ghostdevv) [reply documentation/docs/40-best-practices/20-seo.md:137]: tysm

**Comment by benmccann** [comment]
Thank you! I might wait a few days to review this to see if https://github.com/leeoniya/dropcss/pull/64 gets accepted first

File: documentation/docs/40-best-practices/20-seo.md [hunk 2]
```diff
@@ -106,21 +106,64 @@ export default config;
...
+
+	return await resolve(event, {
+		transformPageChunk: ({ html, done }) => {
+			buffer += html;
+
+			if (done) {
+				let css = '';
+				const markup = amp
+					.transform(buffer)
+					.replace('⚡', 'amp')
```
Comment (dummdidumm) [inline documentation/docs/40-best-practices/20-seo.md:154]:
```suggestion
					.replace('⚡', 'amp') // dropcss can't handle this character
```

**Review by dummdidumm — APPROVED** [review]
Thank you!

**Pull request merged**